ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Connection.scala

```diff
@@ -16,3 +23,41 @@ object Connection {
     Transactor
       .fromDriverManager[IO](jdbcDriver, jdbcUrl, username, password)(IO.ioConcurrentEffect(cs), cs)
 }
+
+object ConnectionPool {
+
+  // below values are hardcoded for now, can be passed as props later.
+  final val MinIdle = 8
+  final val IdleTimeout = 10000
+  final val PoolSize = 10
```

Comment:
Definitely in favor of exposing these via a cli flag. If nothing else, it makes it easier to experiment with different parameters.